src/components/TextAreaChat/styles.ts

```diff
@@ -16,7 +16,7 @@ export const S = StyleSheet.create({
     borderRadius: 50,
     height: 30,
     width: 30,
-    backgroundColor: COLORS.gray9,
+    backgroundColor: colors.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated correctly according to the new color scheme.
```diff
- backgroundColor: COLORS.gray9,
+ backgroundColor: colors.gray1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->